tools/vsimporter/src/SBProject.cpp

```diff
@@ -163,6 +163,17 @@ bool SBProject::checkTargetCycle(const PBXTarget* target)
   }
 }
 
+bool SBProject::containsPackagebleProject()
+{
+  for (auto target : m_existingTargets) {
+    TargetProductType productType = target.second->getProductType();
+    if (productType == TargetStaticLib || productType == TargetBundle || target.first->getType() == "PBXAggregateTarget") {
```

Comment:
This seems wrong. I think we should try to figure out the "end product" instead of just containing a packagable thing.